app/src/org/commcare/android/framework/CommCareActivity.java

```diff
@@ -117,7 +125,15 @@ protected void onCreate(Bundle savedInstanceState) {
         
         mGestureDetector = new GestureDetector(this, this);
     }
-    
+
+    protected void restoreLastQueryString(String key) {
+        SharedPreferences settings = getSharedPreferences(CommCarePreferences.ACTIONBAR_PREFS, 0);
+        lastQueryString = settings.getString(key, null);
+        if (BuildConfig.DEBUG) {

```

Comment:
I'm missing an import for BuildConfig. Which import is the correct one? dalvik, mapviewballoons, chartdemo?